source/Halibut.Tests/WhenCancellingARequestForAPollingTentacle.cs

```diff
@@ -39,7 +37,8 @@ await using (var clientAndService = await clientAndServiceTestCase.CreateTestCas
                 {
                     var doSomeActionService = clientAndService.CreateAsyncClient<IDoSomeActionService, IAsyncClientDoSomeActionServiceWithOptions>();
 
-                    await AssertAsync.Throws<OperationCanceledException>(() => doSomeActionService.ActionAsync(halibutProxyRequestOptions));
+                    (await AssertAsync.Throws<Exception>(() => doSomeActionService.ActionAsync(halibutProxyRequestOptions)))
```

Comment:
Just wondering, could we just check `Throws<ConnectingRequestCancelledException>` instead of `.And.Should().Match(x => x is ConnectingRequestCancelledException);`?

Reply:
Whatever the outcome, there are other instances of this type of check below as well

Reply:
Yes, good catch. These can be simplified now. It's evolved a bit, and started out having to test a type and property of the exception but that's all changed.

Reply:
Done